python/sglang/srt/model_executor/model_runner.py

```diff
@@ -762,7 +766,7 @@ def init_memory_pool(
             self.req_to_token_pool = ReqToTokenPool(
                 size=max_num_reqs + 1,
                 max_context_len=self.model_config.context_len + 4,
-                device=self.device,
+                device=self.device if self.device != "hpu" else "cpu",
```

Comment:
More: dose hpu platform use cpu as device_type for dispatching? We need some principles to distinguish them, otherwise there will be ambiguity in the architecture